src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -170,6 +178,144 @@ std::vector<hoi4::EquipmentVariant> DetermineActiveVariants(const std::vector<ho
 }
 
 
+std::optional<hoi4::Unit> MakeTemplate(const hoi4::DivisionTemplate division, std::vector<hoi4::Battalion>& battalions)
+{
+   hoi4::UnitCount required = division.GetBattalions();
+   for (const auto& [ut, str]: division.GetSupport())
+   {
+      required[ut] += str;
+   }
+   float total = 0.0F;
+   for (const auto& [ut, str]: required)
+   {
+      total += str;
+      float found = 0.0F;
+      for (const auto& battalion: battalions)
+      {
+         if (battalion.GetType() != ut)
+         {
+            continue;
+         }
+         found += battalion.GetStrength();
+      }
+      if (found < str)
+      {
+         return {};
+      }
+   }
+   if (total < tolerance)
+   {
+      // Avoid an infinite loop of making divisions that don't require any strength.
+      return {};
+   }
+   int equipment = 100;
+   int location = 11666;  // Vienna.
```

Comment:
You should substitute in the actual country capital. Or maybe leave it unset, if that could allow people to deploy anywhere.

Reply:
Leaving it unset disappears the division, so I now default to the capital.